client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -130,6 +130,7 @@ struct ConcordClientConfig {
   SubscribeConfig subscribe_config;
   StateSnapshotConfig state_snapshot_config;
   uint64_t metrics_dump_interval_ms = 600;
+  std::uint32_t max_reply_buffer_size;
```

Comment:
Where are you going to use it?

Reply:
its being used in reques_service.cpp, through client_->getMaxReplyBufferSize();

Reply:
Previously you did not have max_reply_size in req_config. Where do you use req_config.max_reply_size?

Reply:
previously it was not configurable, when we create requestconfig object in https://github.com/vmware/concord-bft/blob/master/client/clientservice/src/request_service.cpp:96, it was having default 1MB   because of 
 https://github.com/vmware/concord-bft/blob/master/client/bftclient/include/bftclient/config.h:79, which was not getting overridden.